commands/orchestrationcommands/stackcommands/adopt.go

```diff
@@ -0,0 +1,140 @@
+package stackcommands
+
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/rackspace/rack/commandoptions"
+	"github.com/rackspace/rack/handler"
+	"github.com/rackspace/rack/internal/github.com/codegangsta/cli"
+	osStacks "github.com/rackspace/rack/internal/github.com/rackspace/gophercloud/openstack/orchestration/v1/stacks"
+	"github.com/rackspace/rack/internal/github.com/rackspace/gophercloud/rackspace/orchestration/v1/stacks"
+	"github.com/rackspace/rack/util"
+)
+
+var adopt = cli.Command{
+	Name:        "adopt",
+	Usage:       util.Usage(commandPrefix, "adopt", "--name <stackName> --adopt-file <adoptFile>"),
+	Description: "Creates a stack from existing resources",
```

Comment:
Maybe we can say something here about how you can call the `abandon` command, store the result in a file (with `>>`), and then use it for `adopt-file`.